stdlib/public/core/IntegerTypes.swift.gyb

```diff
@@ -14,10 +14,9 @@
 # Utility code for later in this template
 #
 
-from SwiftIntTypes import all_integer_types, int_max_bits, should_define_truncating_bit_pattern_init
-from SwiftFloatingPointTypes import getFtoIBounds
+import SwiftIntTypes
+import SwiftFloatingPointTypes 
```

Comment:
I didn’t need these changes back in January. Are you sure they are needed?

(The change with the `//` is needed, only the `import` ones werent't needed)